content/articles/2019-10-styler-1.2.0.Rmarkdown

```diff
@@ -0,0 +1,228 @@
+---
+title: "styler 1.2.0"
+author: "Lorenz Walthert"
+date: '2019-10-27'
+description: |
+  Since the initial CRAN submission almost two years ago, many bugs were fixed
+  and new features added, including alignment detection, improved handling of 
+  `{{` and `tryCatch()` expressions and more.
+photo:
+  author: Heng Films
+  url: https://unsplash.com/photos/mpdIPhYqZ4Y
+slug: styler-1.2.0
+categories: ["package"]
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px", 
+  eval = FALSE
+)
+```
+
+The 
+[initial version of styler](https://www.tidyverse.org/articles/2017/12/styler-1.0.0/) 
+was released on CRAN in December 2017. Since then,
+five new versions were released and styler was improved substantially, but many
+improvements were related to special cases or even edge cases. These are all
+contained in the [change log](https://styler.r-lib.org/news/index.html). Here,
+we want to focus on a few changes up to version 1.2.0 that we believe are the 
+most relevant. You can install the latest stable version from CRAN with
+`install.packages("styler")`.
+
+# Alignment detection
+
+styler can finally detect aligned code and keep it aligned! For example, the 
+following code won't be modified by styler:
+
+```{r}
+call(
+  some_arg = 123,
+  more     = "x"
+)
+```
+
+Although quite intuitive, you may need to tweak your aligned code a bit to match
```

Comment:
This feels like you're giving mixed messages — is it intuitive or do I need to read a vignette about it?

Reply:
It's intuitive in the example above but for calls like `tibble::tribble()` with multiple columns there are different ways to align the code and we don't support all of them to keep the complexity low. So I think we should anyways include the link so people can know where to look for advice in case styler does not recognize the alignment. @batpigandme can you check the vignette and give feedback on that?

Reply:
https://styler.r-lib.org/articles/detect-alignment.html